release-notes/VERSION-2.x

```diff
@@ -4,6 +4,14 @@ Project: jackson-databind
 === Releases === 
 ------------------------------------------------------------------------
 
+2.12.8 (11-Oct-2022)
+
+#3582: Add check in `BeanDeserializer._deserializeFromArray()` to prevent
```

Comment:
If there is a new jackson-databind release, it will likely be 2.12.7.1

Reply:
Np, I can change these things. But yes, it will be 2.12.7.1.